### PR TITLE
Adding terms and conditions modal

### PR DESCRIPTION
**Pull request by rodrigohdiaz**
## Proposed Changes
1. Add a term and conditions modal

## Additional Info.
Please read the terms and conditions. I also made the button for the T's & C's, I wasn't sure of implementations.

## Screenshots (optional
Screen Shot 2021-09-20 at 11 44 45 PM
)

### CODE REVIEW

**Review by waltzaround — CHANGES_REQUESTED** [review]
Loks great - only thing I'd recommend would be adding `line-height: 1.5 to that specific modal` and a back button at the bottom of the page

**Comment by noway** [comment]
Can we replace the button with just a link (like other footer links)? I think that way it would look way nicer :-)

File: src/TermsAndConditionsModal.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        </a>
+        <br />
+        <br />
+      </p>
+      <h3>Disclaimer</h3>
+      <p>
+        We work hard to be as updated as possible, but we do not guarantee the
+        correctness of the data we provide. We are a group of individuals
+        working to bring this information and facilitate people to book their
+        vaccine against COVID-19 in Auckland via the 'bookmyvaccine.nz' website.
```
Comment (olafwrieden) [inline src/TermsAndConditionsModal.tsx:90]: @rodrigohdiaz please reword this section to something along the lines of:

"We are committed to serving the most up-to-date vaccine booking availabilities but cannot guarantee the correctness of all booking slots. We are a team of individuals working to facilitate the vaccine booking process across New Zealand by enabling users to browse vaxx.nz intuitively, locate the desired vaccine availability, and directing the user to the official bookmyvaccine.nz website where they complete their booking.
As long as the website, its information and services are provided free of charge, we will not be liable for any loss or damages of any nature."

File: src/TermsAndConditionsModal.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+          marginRight: "1rem",
+        }}
+      >
+        Welcome to vaxx.nz These terms and conditions outline the rules and
+        regulations for the use of vaxx.nz. By accessing this website we assume
+        you accept these terms and conditions. Do not continue to use if you do
+        not agree to take all of the terms and conditions stated on this page.
+        The following terminology applies to these Terms and Conditions, Privacy
+        Statement and Disclaimer Notice and all Agreements: “You” refers to you,
+        the person log on this website and compliant to the Company's terms and
```
Comment (olafwrieden) [inline src/TermsAndConditionsModal.tsx:52]: "the person using this website and compliant to the site's terms and conditions."

File: src/TermsAndConditionsModal.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        refers to the website provided by the Ministry of Health of New Zealand
+        where "you" will be taken, once clicked on the link, to book your
+        vaccine.
+      </p>
+      <br />
+      <h3>Cookies</h3>
+      <p>
+        We employ the use of cookies. By accessing vaxx.nz, you agreed to use
+        cookies in agreement with vaxx.nz's Privacy Policy. Most interactive
+        websites use cookies to let us enhance the user's experience. Cookies
```
Comment (olafwrieden) [inline src/TermsAndConditionsModal.tsx:63]: "We use cookies to enhance..."

File: src/TermsAndConditionsModal.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+        are used by our website to enable the functionality of certain areas to
+        make it easier for people visiting our website.
+      </p>
+      <br />
+      <h3>Privacy Policy</h3>
+      <p>
+        We do not collect any information, neither we collect any information
+        when making a booking for the vaccine through the Ministry of Health NZ.
+        We shall not be held responsible for any breach to the bookmyvaccine.nz
+        website. bookmyvaccine.nz privacy statement: <br />
```
Comment (olafwrieden) [inline src/TermsAndConditionsModal.tsx:73]: ..can be found at this link

File: src/TermsAndConditionsModal.tsx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+        working to bring this information and facilitate people to book their
+        vaccine against COVID-19 in Auckland via the 'bookmyvaccine.nz' website.
+        As long as the website and the information and services on the website
+        are provided free of charge, we will not be liable for any loss or
+        damage of any nature.
+      </p>
+      <br />
+      <h3>Content Liability</h3>
+      <p>
+        We do not ensure that the information on this website is correct, we do
```
Comment (olafwrieden) [inline src/TermsAndConditionsModal.tsx:98]: With the rapidly changing environment, we cannot guarantee up-to-date information on this website.

**Review by olafwrieden — CHANGES_REQUESTED** [review]
Request to change wording in numerous locations to align with Vaxx.nz messaging.

**Comment by olafwrieden** [comment]
@rodrigohdiaz please see the changes requested above and commit them to the branch to update this PR. We'd really love to ship the T&C's now that we are expanding our reach with yesterday's widget announcement. Let us know if you need a hand.

File: src/TermsAndConditionsModal.tsx (hunk 1 above)
Comment (rodrigohdiaz) [reply src/TermsAndConditionsModal.tsx:90]: Ok, working on it

**Comment by wseagar** [comment]
I think adding an ad-hoc terms and conditions written not reviewed by a lawyer is going to open us up to a bunch of issues. 

Here is my proposed terms and conditions, privacy policy and cookie policy based on a template on https://accord.so by @mkyujung. I have edited these to be relevant for vaxx.

**Comment by wseagar** [comment]
**Terms and Conditions**

Effective as of September 21, 2021.

This agreement constitutes a binding legal contract between vaxx.nz(“vaxx.nz”) (sometimes referred to as “we”, “us” or “our”) and you, with respect to your use of https://vaxx.nz and all websites, applications and communications that post a link to this agreement (collectively, the “Site”)(“Terms and Conditions”).

We reserve our right to change these Terms and Conditions in the future. Except as explicitly stated otherwise, your continued use of the Site will constitute deemed acceptance of our updated Terms and Conditions. 

### Site
(a) We may from time to time, at our sole and absolute discretion and without notice to you, update the Site (or any part of it).
(b) We will not be liable to you or any third party for any modification, variation, interruption, suspension or discontinuation of the Site.  The Site may provide links to third party websites that are not owned, managed or controlled by us. You expressly acknowledge and agree that we are not responsible for the content of those third party websites. You are encouraged to read that third party’s terms and conditions, privacy policy and any other document that governs your relationship with that third party. 
(c) The Site uses cookies in accordance with our Cookies Policy, which can be found here: `${cookiePolicyURL}`.

### Prohibited Uses
Unless explicitly agreed by us, you acknowledge and agree that you may not use our Site for the following purposes: 
 - use the Site in any way that violates any applicable local, national or international law or regulation
 - impersonate or attempt to impersonate any person or entity
 - engage in any conduct that restricts or inhibits anyone’s use or enjoyment of the Site
 - attempt to gain unauthorized access to, interfere with, damage or disrupt any parts of the Site
 - use the Site in any manner that could disable, overburden, damage, or impair the Site
 - attempt to interfere with the proper working of the Site in any way whatsoever


### Specifications
(a) We may, from time to time, set certain minimum specifications required to access our Site to ensure all users have the best possible experience. You are solely responsible to obtain, keep and maintain all equipment and other software that meets our minimum specifications to enable you to have the best possible experience of our Site. 
(b) We may from time to time, at our sole discretion and without notice to you, make variations, modifications, alterations or updates to our Site (“Enhancements”). These Enhancements may be made to improve our Site to you or to comply with relevant legal requirements.
(c) We will use reasonable endeavors to notify you of any scheduled disruptions to our Site, including those due to any Enhancements. However, regardless of whether a disruption is scheduled or not, we are not liable to you or any third party for any loss or damage caused by any disruption to our Site.

### Copyrights and Trademarks
(a) Except as expressly stated otherwise, you acknowledge and agree that the original content on the Site, and the software, features and functionality comprising the Site are the exclusive property of Vaxx.nz (the “Materials”) and its licensors.
(b) You agree you will not copy, reproduce, create derivative works from, transmit or distribute the Materials in any way without our prior written consent. 

### Limitation of Liability, Indemnities and Warranties
(a) Except as expressly provided by law, we shall not be liable to you or any third parties for any loss, damage, expenses or any other liability arising directly or indirectly from the performance of our services to you.  To the fullest extent permitted by law, all warranties or conditions implied by statute, at law, by trade, custom or otherwise are excluded. 
(b) You acknowledge and agree that you assume sole and entire responsibility for, and indemnify and hold us harmless from, any and all claims, liabilities, losses, expenses, responsibilities and damages by reason of any claim, proceedings, action, liability or injury arising out of or as a result of (i) your conduct in relation to these Terms and Conditions; (ii) your use of any material, advice or other results of the services provided to you; (iii) your relations with your clients and/or other third parties; or (iv) any breach of these Terms and Conditions by you. 

### Term and Termination
(a) These Terms and Conditions will remain in full force and effect while you use the Site.
(b) We may, without notice to you, immediately stop providing any services to you at our sole discretion for any reason whatsoever. 
(c) Where these Terms and Conditions are terminated in accordance with these Terms and Conditions, they will terminate without prejudice to any rights either party may have had against the other prior to termination. Further, all provisions of which by their nature should survive termination shall survive termination, including, without limitation: (i) all intellectual property you have granted us under these Terms and Conditions; and (ii) your obligation to pay any fees due to us at the time of termination; and (iii) warranty disclaimers, indemnity and limitations of liability. 

### Assignment
(a) These Terms and Conditions will bind and inure for the benefit of the parties, including their respective successors, permitted assigns and legal representatives.
(b) We may, without notice to you, immediately stop providing any services to you at our sole discretion for any reason whatsoever. 
(c) Provided your rights are not affected under these Terms and Conditions, we will be permitted to assign our interest in these Terms and Conditions without prior notice to you. However, you may not assign, transfer, novate or other dispose of your rights and obligations under these Terms and Conditions without our express written approval (which we may withhold in our sole discretion). 

### Privacy
You acknowledge that you have read and understood the terms of vaxx.nz's Privacy Policy, which can be found here: `${privacyPolicyUrl}`. You agree that we may use your information (including disclosure to third parties) in accordance with the terms of our Privacy Policy.

### Non-Waiver
Our failure to exercise, or delay in exercising, our rights under these Terms and Conditions does not operate as a waiver of that right.

### Exclusion of Implied Relationships

Nothing in these Terms and Conditions shall be deemed or construed to constitute any party a partner, agent, representative, employer or employee of another party or to create any trust or commercial partnership unless specifically otherwise provided. We do not owe you any duty of good faith under these Terms and Conditions. 

### Severability

If any part of these Terms and Conditions are held to be illegal, invalid, or unenforceable, then that part shall be deemed deleted and shall not affect the validity and enforceability of the remaining provisions of these Terms and Conditions. 

### Entire Agreement
These Terms and Conditions constitute the entire agreement between the parties with respect to the Company’s services and supersede all previous versions of the Terms and Conditions, understandings, arrangements, agreements, and communications, whether verbal or written, between the parties.

### Applicable Law and Jurisdiction
These Terms and Conditions shall be governed by and construed in accordance with N/A law. Both parties agree to submit to the jurisdiction of the courts of N/A with respect to any claim or dispute arising out of these Terms and Conditions.

**Comment by wseagar** [comment]
**Privacy Policy**

This privacy policy sets out how vaxx.nz (“vaxx.nz”) collects, processes and uses your Personal Information through your use of our Services.

### Definitions

**Anonymized Data** means anonymous information collected by us through your use of this Website or our Services for statistical and/or demographic purposes. Anonymized Data may be derived from Personal Information.

**Data Processor** means each of the following: Github; Google Analytics; and Netlify.

**DPO** means Data Protection Officer, and is the person specifically appointed by us to deal with your enquiries with respect to this privacy policy and your Personal Information. 

**Personal Information** means any identifying information about you. This includes, but is not limited to, the following: cookies and usage data.

**Services** means all services provided by vaxx.nz to you, including (but not limited to) this Website and the following: Delivery of product and service.

**Website** means vaxx.nz and all websites, applications and communications that post a link to this Privacy Policy.

### Personal Information Collection

We only collect and use Personal Information to the extent necessary to provide you with the Services. We collect Personal Information for the Website to provide you with a better online experience.

We also provide the following services: 
**Delivery of product and service**

To deliver the Services, fulfill and manage customer orders and provide customer support.


### Personal Information Retention

We keep Personal Information for the shortest time necessary to provide you with Services and to meet all our legal and compliance obligations. To determine this retention period, we take into account (i) the nature of the Personal Information gathered; and (ii) the nature of our legal and compliance obligations. All Personal Information no longer required by us is destroyed and/or erased.

### Anonymized Data

We may use, collect and share Anonymized Data for any purpose at our discretion. Anonymized Data cannot be used to reveal your identity (either directly or indirectly). We may retain all Anonymized Data indefinitely. 
Anonymized Data includes:

 - Device data: This includes information about the device used to interact with our Services.
 - Location data: This includes geolocation data (based on your IP address).
 - Log and usage data: This includes usage and performance information with respect to our Services.


### Your Rights

With respect to Personal Information we hold, you have the following rights:
 - **Access**: You may request from us access to your data that we hold on you. 
 - **Rectification**: If the data we hold on you is inaccurate, you may request that we correct it. If the data we hold is incomplete, you may request that we complete it. 
 - **Erasure**: Subject to certain conditions, you may request that we erase all of the data we hold on you. 
 - **Restrictions**: Subject to certain conditions, you may request that we restrict the processing of data we hold on you.
 - **Portability**: Subject to certain conditions, you may request that we transfer all the data we hold on you to a third party (including yourself).
 - **Objections**: Subject to certain conditions, you may object to us processing the data we hold on you.

### Data Processors
We engage Data Processors to assist us in providing you with Services. Data Processors are contractually obligated to us, and must:

- comply with, and act only on, our instructions with respect to the data that they process on our behalf (we, of course, give our instructions in accordance with this privacy policy);
- notify us if they believe we are acting in breach of the GDPR (where applicable);
- first obtain our consent prior to sub-contracting data processing services;
- comply with their obligations under the GDPR (where applicable).

### Social Media

You may choose to engage with our Services through a variety of social media sites and platforms, for example Facebook, Instagram and Twitter. When you interact with our Services through these social media platforms, you may allow us to receive Personal Information and other Anonymized Data on how you are interacting with our Services. This will depend on your privacy settings on each social media site and platform.

The data we are provided by you through your interaction with our Services on social media sites and platforms will be used by us subject to the terms of this privacy policy. Depending on your social media site’s privacy settings, your Personal Information may become public to a community of users if you choose to engage with our Services through these platforms.

For further information on how you can customize your privacy settings on social media sites and how those sites handle your privacy, please refer to the privacy policy of each particular social media site. 

### Minors
We do not knowingly collect information on children under the age of 16 (“**Child**”).  If you become aware that a Child has provided us with Personal Information, please contact us. If we become aware that we have collected Personal Information from a Child without their parent's verifiable authorization to access our Services, we will take steps to remove that information from our servers.

### Third Party Links
Our website contains links to other third-party websites not owned or managed by vaxx.nz. This privacy policy applies to this website only. If you click a link to a third-party website, the privacy policy of that website will apply. We highly recommend that you read the privacy policies of other websites as they may be different from ours.

### Updates
We are constantly reviewing our privacy policy and procedures to ensure that they meet best practices. This policy was last updated on September 21, 2021.

### Contact us
If you have any questions or suggestions about our privacy policy or want to know more information about Personal Information we hold, please contact us at info@vaxx.nz.

### Complaints
If you believe that we have breached our obligations to you under this privacy policy or relevant privacy laws, please contact us at info@vaxx.nz. 
If you are not satisfied with our response, you have the right to lodge a complaint at any time to your relevant authority.

**Comment by wseagar** [comment]
**Cookies Policy**

Cookies are small files that are downloaded onto your device by your web browser when you access any website or app that posts a link to this policy (collectively “the Sites”). Different types of cookies provide us with different information, all of which are used to provide you with a better experience on our Sites and provide us with greater information on users to make further improvements. 

### Types of Cookies
**Preference Cookies**: Preference cookies allow us to remember your preferences and past actions.

### First and Third-Party Cookies
**First-Party Cookies**: First-party cookies are controlled by us.
**Third-Party Cookies**: Third-party cookies are controlled by our third-party advertisers. A list of our third-party advertiser affiliates is contained in our privacy policy. 

Depending on the cookie, our Sites may use both first-party and third-party cookies. 

### Control of Cookies

Most browsers are set to accept cookies by default. However, you can remove or reject cookies in your browser’s settings. This could affect the availability and functionality of the Sites. For more information on how to control cookies, check your browser or device’s settings for how you can control or reject cookies, or visit the following links:

**Google Chrome**: https://support.google.com/chrome/answer/95647?co=GENIE.Platform%3DDesktop&hl=en
**Apple Safari**: https://support.apple.com/en-nz/guide/safari/sfri11471/mac
**Microsoft Edge:** https://support.microsoft.com/en-us/windows/microsoft-edge-browsing-data-and-privacy-bb8174ba-9d73-dcf2-9b4a-c582b4e640dd
**Microsoft Internet Explorer**: https://support.microsoft.com/en-us/windows/delete-and-manage-cookies-168dab11-0753-043d-7c16-ede5947fc64d
**Mozilla Firefox**: https://support.mozilla.org/en-US/kb/enhanced-tracking-protection-firefox-desktop?redirectslug=enable-and-disable-cookies-website-preferences&redirectlocale=en-US
**Opera**: https://help.opera.com/en/latest/
**Android (Chrome)**: https://support.google.com/chrome/answer/95647?co=GENIE.Platform%3DAndroid&hl=en&oco=1
**iPhone and iPad (Chrome)**: https://support.google.com/chrome/answer/95647?co=GENIE.Platform%3DiOS&hl=en&oco=1
**iPhone and iPad (Safari)**: https://support.google.com/chrome/answer/95647?co=GENIE.Platform%3DAndroid&hl=en&oco=1

### How Long are Cookies Stored?

**Session Cookies**: Session cookies are temporary and only retain information on your activities on our Sites for as long as you remain on our Sites. Session cookies are deleted when your web browser is closed.
**Permanent Cookies**: Permanent cookies remain on your device until they are deleted or on the cookie’s expiration date.
Depending on the cookie, our Sites use both session and permanent cookies.

**Cookies and Personal Information** 
Where cookies are used to collect your Personal Information, or other Anonymized Data (as these terms are defined our privacy policy), they will be used, stored and destroyed in accordance with our privacy policy.

**Comment by wseagar** [comment]
@rodrigohdiaz Sorry for adding a bunch more work on you and changing the ticket half way though/giving mixed messaging , really appreciate your eagerness to contribute!

I didn't realise we had an open ticket for t&c's I would have added the content there before you started work. 

I've done some changes on your branch for how this should work. I've added 3 routes and added the content in markdown format I mentioned above to them.

"/terms-and-conditions"

"/privacy-policy"

"/cookies-policy"

If you're feeling especially motivated you could move it out of markdown and into react/html but that's not a requirement, (we could do this in another ticket). I only did it this way because it was quick and I had the content formatted(ish) in markdown.

Could I get you to

1. Add links in the footer to these routes
2. Remove the T&C modal
3. Tidy up the markdown styling I've checked in (optional)
4. Tidy up the styling on these routes (background color and border to match the rest of the site) (optional)

Only 1 & 2 will be required to get this merged.

@MinhPhu0304 / @oeed / @olafwrieden  For a future issue could we get a ticket to move the routes to be lazy loaded so this markdown parser I've added isn't included in the main site bundle? Not a priority now.

**Comment by waltzaround** [comment]
In retrospect, agree with @wseagar

**Comment by noway** [comment]
Makes sense to me. Yep, let's go with what Will said.

File: src/App.tsx [hunk 6]
```diff
@@ -43,6 +45,21 @@ function App() {
       <div className="App">
         <Header />
         <Switch>
+          <Route path="/terms-and-conditions">
+            <div className={"big-old-container"}>
+              <ReactMarkdown>{terms}</ReactMarkdown>
```
Comment (wseagar) [inline src/App.tsx:50]: Could you replace these 

```
<ReactMarkdown>...</ReactMarkdown>
```

blocks with your new components?

ie

```
<TermsAndConditions />
```

@gpoliko

**Review by noway — APPROVED** [review]
image
There's just 1 little wrinkle: this text blob doesn't have line breaks. But it's all good by me, good to merge otherwise 👍🏻

**Review by waltzaround — APPROVED** [review]
Good enough for v1 - Let's clean this up in a future PR 😅

**Review by olafwrieden — APPROVED** [review]
can definitely do with a clean up but looks alright to me.

**Pull request merged**